libr/arch/p/arch_riscv.c

```diff
@@ -2,13 +2,14 @@
 
 #include <r_lib.h>
 #include <r_asm.h>
-#include <r_anal.h>
+#include <r_arch.h>
 #include "../../asm/arch/riscv/riscv-opc.c"
 #include "../../asm/arch/riscv/riscv.c"
```

Comment:
its never a good idea to include a c file, as long as those should be dynamically linked as separate objects. but its fine for now. we will do more reviews